src/components/StructuredList/StructuredList-story.js

```diff
@@ -13,103 +14,124 @@ import {
-            vulputate nisl a porttitor interdum.
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  ))
-  .add('Selection', () => (
-    <StructuredListWrapper selection border>
-      <StructuredListHead>
-        <StructuredListRow head>
-          <StructuredListCell head>{''}</StructuredListCell>
-          <StructuredListCell head>ColumnA</StructuredListCell>
-          <StructuredListCell head>ColumnB</StructuredListCell>
-          <StructuredListCell head>ColumnC</StructuredListCell>
-        </StructuredListRow>
-      </StructuredListHead>
-      <StructuredListBody>
-        <StructuredListRow label htmlFor="row-1">
-          <StructuredListInput
-            id="row-1"
-            value="row-1"
-            title="row-1"
-            name="row-1"
-            defaultChecked
-          />
-          <StructuredListCell>
-            <Icon
-              className="bx--structured-list-svg"
-              icon={iconCheckmarkSolid}
-              description="select an option"
+  .add(
+    'Simple',
+    withInfo({
+      text: `
+        Structured Lists group content that is similar or related, such as terms or definitions.
+      `,
+    })(() => (
+      <StructuredListWrapper>
+        <StructuredListHead>
+          <StructuredListRow head>
+            <StructuredListCell head>ColumnA</StructuredListCell>
+            <StructuredListCell head>ColumnB</StructuredListCell>
+            <StructuredListCell head>ColumnC</StructuredListCell>
+          </StructuredListRow>
+        </StructuredListHead>
+        <StructuredListBody>
+          <StructuredListRow>
+            <StructuredListCell noWrap>Row 1</StructuredListCell>
+            <StructuredListCell>Row 1</StructuredListCell>
+            <StructuredListCell>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dui
+              magna, finibus id tortor sed, aliquet bibendum augue. Aenean posuere
+              sem vel euismod dignissim. Nulla ut cursus dolor. Pellentesque
+              vulputate nisl a porttitor interdum.
+            </StructuredListCell>
+          </StructuredListRow>
+          <StructuredListRow>
+            <StructuredListCell noWrap>Row 2</StructuredListCell>
+            <StructuredListCell>Row 2</StructuredListCell>
+            <StructuredListCell>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dui
+              magna, finibus id tortor sed, aliquet bibendum augue. Aenean posuere
+              sem vel euismod dignissim. Nulla ut cursus dolor. Pellentesque
+              vulputate nisl a porttitor interdum.
+            </StructuredListCell>
+          </StructuredListRow>
+        </StructuredListBody>
+      </StructuredListWrapper>
+    ))
+  )
+  .add(
+    'Selection',
+    withInfo({
+      text: `
+        Structured Lists with selection allow a row of list content to be selected.
```

Comment:
For the `default` text description, I copied the the description here: http://carbondesignsystem.com/components/structured-list/code

But for this `Selection` variation, I wasn't sure what to write. 🤔 Please let me know if you'd like for me to change this description wording.